cmd/grpcui/grpcui.go

```diff
@@ -444,13 +451,20 @@ func main() {
 			logInfof("%s %s %s %d %dms %dbytes", r.RemoteAddr, r.Method, r.RequestURI, cs.code, millis, cs.size)
 		})
 	}
+	if *basePath != "/" {
+		handler = http.StripPrefix(*basePath, handler)
```

Comment:
IIRC, this can act funky if the given prefix ends in "/". For example, if we used "/grpcui/" as the prefix, then "/grpcui" results in a 404 and even "/grpcui/" is weird because it strips the leading "/" from the URI (which I think means it won't correctly match any URIs in the underlying mux).

So I think we need to remove any trailing slash.

```suggestion
		handler = http.StripPrefix(strings.TrimSuffix(*basePath, "/"), handler)
```